css/style.css

```diff
@@ -0,0 +1,1140 @@
+  .main__container {
+    margin-top: 465px;
+  }
+
+  .main__row {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .main__container-second {
+    margin-right: 0;
+  }
+}
+
+@media (max-width: 480px) {
+  .header {
+    padding: 0;
+  }
+
+  .header__logo {
+    margin: 0;
+    margin-left: -7px;
+  }
+
+  .header__img-wordmark {
+    width: 138px;
+    height: 42px;
+    left: 2px;
+    margin-left: 6px;
+  }
+
+  .header__img-burger {
+    width: 24px;
+    height: 15px;
+    margin-right: 19px;
+    margin-top: 6px;
+  }
+
+  .main {
+    margin-top: 110px;
+  }
+
+  .main__img {
+    margin-top: 19px;
+    width: 450px;
+    height: 210px;
+    left: -20px;
+  }
+
+  .main__title {
+    margin-top: -106px;
+    margin-left: -6px;
+    font-size: 50px;
+    margin-bottom: 5px;
+    padding: 0;
+    justify-content: center;
+  }
+
+  .main__description {
+    justify-content: center;
+    font-size: 10px;
+    text-align: center;
+    margin-left: -5px;
+    margin-bottom: -20px;
+    margin-top: -31px;
+  }
+
+  .main__container {
+    flex-direction: column;
+    padding: 0 36px;
+    justify-content: center;
+    margin-top: 264px;
+  }
+
+  .main__container-first {
+    width: 100%;
+  }
+
+  .main__container-second {
+    width: 100%;
+  }
+
+  .main__row {
+    margin-bottom: 43px;
+    margin-right: 0;
+    margin-left: 0;
+  }
+
+  .main__row-title {
+    font-size: 27px;
+    margin: -8px 14px 0 0;
+  }
+
+  .button {
+    margin: 0;
+    justify-content: center;
+    width: 160px;
+    margin-left: 25px;
+  }
+
+  .main__button {
+    padding: 11px 6px;
+    height: 40px;
+    margin-top: 22px;
+  }
+
+  .main__img-vector {
+    margin-top: 5px;
+    width: 44.14px;
+  }
+
+  .main__img-stroke {
+    width: 33.83px;
+    margin-right: -2px;
+  }
+
+  .main__img-steering,
+  .main__img-steering-1 {
+    display: none;
+  }
+
+  .main__img-steering-2 {
+    display: inline-block;
+  }
+
+  .main__subtitle-time,
+  .main__subtitle-dist {
+    font-size: 15px;
+  }
+
+  .main__row-desc {
+    font-size: 10px;
+    text-align: left;
+    margin-top: -4px;
+    right: -13px;
+  }
+
+  .main__row-test {
+    margin-left: -42px;
+  }
+
+  .row__test {
+    display: flex;
+    margin-top: 25px;
+    margin-left: 40px;
+  }
+
+  .main__row-text {
+    margin-right: -39.49px;
+    color: #E51937;
+    text-shadow: none;
+    margin-left: 5px;
+    font-size: 13px;
+  }
+
+  .main__arrow {
+    margin: 0 auto;
+  }
+}
+
+/* .container {
+  max-height: 900px;
+} */
+
+.container__title {
+  color: #E51937;
+  margin: 101px 905px 0 80px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 70px;
+  line-height: 85px;
+  z-index: 99;
+}
+
+.container__content {
+  display: flex;
+}
+
+.container__img {
+  display: inline-block;
+  width: 100%;
+  margin-top: -100px;
+  z-index: -1;
+}
+
+.container__img-phone {
+  display: none;
+}
+
+.container__aside {
+  display: flex;
+  flex-direction: column;
+  background-color: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Tesla-Project_Evgen/pullRequest?prid=6071575)